tests/unit/algo/classification/test_deit_tiny.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import pytest
+import torch
+from otx.algo.classification.deit_tiny import (
+    DeitTinyForHLabelCls,
+    DeitTinyForMulticlassCls,
+    DeitTinyForMultilabelCls,
+)
+from otx.algo.utils.support_otx_v1 import OTXv1Helper
+
+
+class TestDeitTiny:
+    @pytest.mark.parametrize(
+        "model_cls",
+        [DeitTinyForMulticlassCls, DeitTinyForMultilabelCls, DeitTinyForHLabelCls],
+    )
+    def test_deit_tiny_for_mulitclass(self, model_cls, mocker):
```

Comment:
```suggestion
    def test_deit_tiny(self, model_cls, mocker):
```

Reply:
It seems that this test's scope is mixed. 
model could be Multiclass, Multilabel, Hlabel, however, testing load_ckpt for only multiclass..?

Reply:
I've changed the test function name, Thanks for the report.
And it seems all Deit model use same load_checkpoint function
![image](https://github.com/openvinotoolkit/training_extensions/assets/106788349/113eba1d-64e1-4b3f-9948-bf278f5a00fc)